nocodb/nocodb
# chore(nc-gui): update to latest vue flow pkg

## Change Summary

- update vue flow to latest pkg version
- use v-model for elements
	- fixes issue where edges aren't rendered on initial erd mount
- check if table exists before creating corresponding edge
- defer fitView transition so it appears correctly
- add min/max zoom values so we can properly fit the view with multiple tables
- move erd element setup logic into a separate file

## Change type


## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of
